changelog/9078.misc.md

```diff
@@ -0,0 +1 @@
+Add test to ensure there are no warnings triggered by the default configuration when training or a loading a model.
```

Comment:
Since we're not testing loading a model - should the last section of the sentence be removed?